PacmanTest.java

```diff
@@ -0,0 +1,155 @@
+package pacman;
+
+import static org.junit.Assert.*;
+import org.junit.jupiter.api.Test;
+
+class PacmanTest {
+
+	Game game;
+	Pacman pacman;
+	
+	public void setGameBoardByStringArray(final String[] board){
+		this.game = new Game(board);
+	}
+	
+	// Pacman: Move
+	
+	// Right
+	
+		@Test
+	public void PacmanRightWall(){
+		// arrange
+		Location l1 = new Location(0, 0), l2 = new Location(0, 1);
+		setGameBoardByStringArray( new String[]{"PW"} );
+		Board.MovingCreature.PACMAN.setDirection(Game.Direction.RIGHT);
+		
+		// action
+		game.move();
+		
+		// assert
+		assertEquals(Board.MovingCreature.PACMAN, this.game.getCreatureAt(l1));
+		assertEquals(Board.Creature.WALL, this.game.getCreatureAt(l2));
+	}
+
+//	@Test
```

Comment:
Why are you submitting commented lines?
This is considered as bad practice, that should be avoided.
if this is a previous code, it's already in source control management (git) and you will be able to recover it from there. otherwise, finish your job completely before pull requesting...